lib/rabbitmq/ffi/ext/field_value.rb

```diff
@@ -40,7 +40,7 @@ def apply(value)
         when ::Symbol; [:utf8,  FieldValueValue.new(Bytes.from_s(value.to_s.encode!(Encoding::UTF_8)).pointer)]
         when ::Array;  [:array, FieldValueValue.new(Array.from_a(value).pointer)]
         when ::Hash;   [:table, FieldValueValue.new(Table.from(value).pointer)]
-        when ::Fixnum; [:i64,       (v=FieldValueValue.new; v[:i64]=value; v)]
+        when 1.class;  [:i64,       (v=FieldValueValue.new; v[:i64]=value; v)]
```

Comment:
This isn't your fault, but I'm a little bit annoyed that Ruby removed the distinction between `Fixnum` and `Bignum`, since `Fixnum` meant a 64-bit integer, which happens to be exactly what I wanted here.

However, would it be possible to use `Integer` here instead of `1.class`?